src/layout/actions/layout.rs

```diff
@@ -821,22 +831,35 @@ impl LayoutTree {
     fn update_container_geo_for_borders(&mut self, node_ix: NodeIndex,
                                         mut geometry: Geometry)
                                         -> Result<Geometry, TreeError> {
+        // Stacked containers need bigger title bars.
+        // This has to be done before borrowing self.tree to use the container
+        let child_count = self.tree.children_of(node_ix).len();
+
         let container = &mut self.tree[node_ix];
 
         match *container {
-            Container::Container { ref mut apparent_geometry,
+            Container::Container { layout,
+                                   ref mut apparent_geometry,
                                    geometry: ref mut actual_geometry,
                                    ref borders, .. } => {
                 *actual_geometry = geometry;
                 if borders.is_some() {
+                    let title_count = match layout {
+                        Layout::Stacked =>
+                            // I don't understand why I have to use this
+                            // instead of just child_count, but seems to work...
+                            (2*child_count-1) as u32,
```

Comment:
This expression is used in a lot of places, is it possible to make this a function somewhere so this piece of code doesn't have to be copied around and then this concept can be given an abstract name to talk about it?

Probably best to place it as a function of `Layout` actually, since it seems dependent on layout (though it's only used for borders, so putting it on borders might make more sense...depends on what you think, I haven't looked into why you need to do that weird double-ing yet. If it's related to borders, put it on borders, if it's a weird quirk of math and how tabbed/stacked works then put it in `Layout` since it's more generic)

Reply:
Yeah, I actually was going to do that, a method `title_bar_size` for the border that did this calculation, but it's not used everywhere, specifically one place I don't remember, it actually calculated the title bar size as expected, just the size times the child count, and if I replaced that with the weird one it could introduce a bug.

So I didn't know how to name the method or where to put it because I don't understand what it's doing.